ACTIONS.md

```diff
@@ -0,0 +1,55 @@
+We use GitHub Actions in order to help automate deployment, check build quality, & other tasks. You can add, edit, & view our GitHub actions by running the following commands in Terminal when in the repo's base directory:
```

Comment:
A level 1 title would be nice (`# Actions` or something more creative).

Reply:
Agreed. Added one of the form "GitHub Actions in Mantle" so as to better communicate this is a page to help understand our workflows.